e2e/davinci-app/main.ts

```diff
@@ -168,7 +171,7 @@ const config = {
    * It returns an unsubscribe function that you can call to stop listening
    */
   davinciClient.subscribe(() => {
-    const node = davinciClient.getClientInfo();
+    const node = davinciClient.getClient();
```

Comment:
This could just be me, but when i read this for some reason my brain thought this would be an async function.

Reply:
Yeah, originally, it was just `clientInfo` or something, but then more methods kept getting added to the API, and it started getting a bit crowded with just having nouns, so I added the verb prefix. These `get*` method don't result in network requests, just "getting" a portion of the existing, stored state.